onelogin_aws_cli/tests/test_configurationFile.py

```diff
@@ -0,0 +1,80 @@
+from io import StringIO
+from unittest import TestCase
+from unittest.mock import patch
+
+from onelogin_aws_cli.configuration import ConfigurationFile
+
+
+class TestConfigurationFile(TestCase):
+
+    def test_can_save_password_username_true(self):
+        str = StringIO()
+        str.write("""[defaults]
+save_password = false
+
+[profile_test]
+save_password = true""")
+        str.seek(0)
+        cfg = ConfigurationFile(str)
+        self.assertTrue(cfg.section("profile_test").can_save_password)
+
+    def test_can_save_password_username_false(self):
+        str = StringIO()
+        str.write("""[defaults]
+save_password = false
+
+[profile_test]
+save_password = false""")
+        str.seek(0)
+        cfg = ConfigurationFile(str)
+        self.assertFalse(cfg.section("profile_test").can_save_password)
+
+    def test_can_save_password_username_xor_1(self):
+        str = StringIO()
+        str.write("""[defaults]
+save_password = false
```

Comment:
Some of these seem repeated.  Did you intend for this to be true / false?

Reply:
These were written before I split the PR for the save-username functionality, and there may be duplicates. Let me have a closer look